Okta.Samples.OpenIDConnect.Console/KeyGenerator.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Okta.Samples.OpenIDConnect
+{
+    public class KeyGenerator
+    {
+        public static string GetUniqueKey(int maxSize)
+        {
+            char[] chars = new char[62];
+            chars =
+            "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890".ToCharArray();
+            byte[] data = new byte[1];
```

Comment:
Could this just be `null` given that the variable gets initialized in the using block?